inference.py

```diff
@@ -221,42 +152,21 @@ def main(params: dict):
     """
     # SET BASIC VARIABLES AND PATHS
     since = time.time()
-
-    debug = get_key_def('debug_mode', params['global'], False)
-    if debug:
-        warnings.warn(f'Debug mode activated. Some debug features may mobilize extra disk space and cause delays in execution.')
-
-    num_classes = params['global']['num_classes']
     task = params['global']['task']
-    num_classes_corrected = add_background_to_num_class(task, num_classes)
-
+    img_dir_or_csv = params['inference']['img_dir_or_csv_file']
     chunk_size = get_key_def('chunk_size', params['inference'], 512)
-    overlap = get_key_def('overlap', params['inference'], 10)
-    nbr_pix_overlap = int(math.floor(overlap / 100 * chunk_size))
```

Comment:
The overlap parameter isn't used anymore, right? 

If so, we should remove it from the README and the template yaml config file.

Reply:
see comment on line #52